src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue

```diff
@@ -23,6 +23,12 @@
 <template>
 	<!-- reactions buttons and popover with details -->
 	<div class="reactions-wrapper">
+		<!-- all reactions button -->
+		<NcButton v-if="reactionsSorted.length > 1"
+			class="reaction-button"
+			@click="showAllReactions = true">
+			{{ t('spreed', 'All') }}
+		</NcButton>
 		<NcPopover v-for="reaction in reactionsSorted"
```

Comment:
This Popover could contain a shortcut, like 'See all reactions'

Reply:
Yes, sound OK by me

Reply:
cc @nextcloud/designers